tomviz/PresetModel.cxx

```diff
@@ -144,8 +144,11 @@ void PresetModel::loadFromFile()
   QJsonArray objects = doc.array();
   for (auto value : objects) {
     QJsonObject obj = value.toObject();
-    QJsonObject nextDefault{ { "name", obj["Name"] },
-                             { "RGBPoints", obj["RGBPoints"] } };
+    QJsonObject nextDefault{
+      { "Name", obj["Name"] },
+      { "ColorSpace", obj["ColorSpace"] },
```

Comment:
Same here. It would be nice to keep the state file and color JSON object using the same case/names so that we only have one convention to worry about in Tomviz.